src/main/java/org/example/App.java

```diff
@@ -1,8 +1,15 @@
 package org.example;
 
+import java.util.Arrays;
+import java.util.stream.Stream;
 
-    public class App {
+public class App {
     public static void main( String[] args ) {
-        System.out.println( "Hello World!" );
+        System.out.println("Sorted word list: ");
+        Arrays.stream(sorted_list(args)).forEach(x -> System.out.println(x));
+    }
+
+    public static Object[] sorted_list(String[] str){
+        return Arrays.stream(str).sorted().toArray();
```

Comment:
Can be replaced with `Arrays.asList(str).stream().sorted().collect(Collectors.toList())`

Also, it's not a good idea to return array of objects. Since under object class there could be any java class.